roles/unicorn_user/templates/defaults.j2

```diff
@@ -1 +1,2 @@
 RAILS_ENV={{ rails_env }}
+SKYLIGHT_AUTHENTICATION={{ skylight_authentication }}
```

Comment:
What happens if the environment variable is not defined? Ruby crash?

And what if the ansible variable is not declared anywhere? Ansible crash?

Reply:
good points

Reply:
When `SKYLIGHT_AUTHENTICATION` is not defined you'll see the following in `log/skylight.log`:

```
[SKYLIGHT] [1.5.0] Unable to start Instrumenter; msg=authentication token required; class=Skylight::ConfigError
```
When its value is empty instead () you see the following in the `log/staging.log`:

```
[SKYLIGHT] [1.5.0] Unable to start, see the Skylight logs for more details
```

Skylight seems pretty robust. So far I've seen meaningful error messages in every situation :muscle: .

Reply:
If the ansible var `skylight_authentication` is not declared you get:

```
(...)
TASK [unicorn_user : Write OFN environment variables defaults] *****************************************************************************
Tuesday 20 March 2018  12:28:35 +0100 (0:00:03.414)       0:00:25.459 ********* 
fatal: [staging.katuma.org]: FAILED! => {"changed": false, "failed": true, "msg": "AnsibleUndefinedVariable: 'foo' is undefined"}
        to retry, use: --limit @/home/pau/dev/ofn-install/playbooks/provision.retry
(...)
```

Reply:
OK for the Skylight warning.

Not so OK for the Ansible failure. Since it's an optional feature some instance could be not interested and shouldn't bother at all about it. `when` conditional in Ansible could help?